apps/cowswap-frontend/src/modules/swap/containers/EthFlow/hooks/useEthFlowActions.ts

```diff
@@ -88,7 +88,9 @@ export function useEthFlowActions(callbacks: EthFlowActionCallbacks): EthFlowAct
     }
 
     const expertModeFlow = () => {
-      Promise.all([isApproveNeeded ? approve(false) : undefined, isWrapNeeded ? wrap(false) : undefined])
+      return Promise.all([isApproveNeeded ? approve(false) : undefined, isWrapNeeded ? wrap(false) : undefined]).then(
+        () => void 0
```

Comment:
nit: a bit hard to read, i think it can be done in more steps instead of a one-liner

Reply:
Added `//` to prevent prettier from breaking line break